pearl/test/integration/integration_tests.py

```diff
@@ -350,6 +351,52 @@ def test_ppo(self) -> None:
             )
         )
 
+    def test_ppo_lstm(self) -> None:
```

Comment:
this test gives quite unstable results for now. Have to check if proper tuning is required or if something's else is wrong.